public/components/ctas/8.html

```diff
@@ -0,0 +1,26 @@
+<aside class="relative overflow-hidden text-gray-300 bg-gray-900 lg:flex justify-center">
+    <div class="w-full p-12 text-center lg:w-2/3 sm:p-16 lg:p-24">
+      <div class="max-w-xl mx-auto lg:ml-0">
```

Comment:
Adding `lg:ml-0` here is causing you to need the `lg:flex justify-center` on the root element for this component.

Removing the previous classes highlighted and `lg:ml-0` solves that.